### PR TITLE
Support for interval

### PR DESCRIPTION
**Pull request by Fredrik-C**
Support for passing interval arguments

### CODE REVIEW

File: Marketstack/Interfaces/IMarketstackService.cs [hunk 1]
```diff
@@ -11,6 +11,6 @@ public interface IMarketstackService
         Task<List<Exchange>> GetExchanges();
         Task<List<Stock>> GetExchangeStocks(string exchangeMic);
         Task<List<StockBar>> GetStockEodBars(string stockSymbol, DateTime fromDate, DateTime toDate);
-        Task<List<StockBar>> GetStockIntraDayBars(string stockSymbol, DateTime fromDate, DateTime toDate);
+        Task<List<StockBar>> GetStockIntraDayBars(string stockSymbol, DateTime fromDate, DateTime toDate, string interval = "1hour");
```
Comment (orshe4) [inline Marketstack/Interfaces/IMarketstackService.cs]: Please move the `interval` to an Enum.

**Review by orshe4 — CHANGES_REQUESTED** [review]
Please upgrade the NuGet version

File: Marketstack/Interfaces/IMarketstackService.cs (hunk 1 above)
Comment (Fredrik-C) [reply Marketstack/Interfaces/IMarketstackService.cs]: Problem is that enums does not allow numbers as a part of their key.

Comment (Fredrik-C) [reply Marketstack/Interfaces/IMarketstackService.cs]: Hm, found a way using description attributes

Comment (Fredrik-C) [reply Marketstack/Interfaces/IMarketstackService.cs]: Ok?

**Pull request merged**